docs/introduction/computational_resources_overview.md

```diff
@@ -1,47 +1,34 @@
-# Shared and Dedicate Resources
+# Overview
 
 In this guide, we will explain some of the main features of the Inductiva API when 
-it comes to making informed decisions about resource allocation, ensuring you get 
-the most out of the Inductiva API for your simulation needs.
-
-Here, you will learn about running your simulation tasks on a shared pool of Virtual
-Machines (WMs), ideal for simple tasks and quick tests. Then, you will examine the 
-options for dedicated resources, including Machine Groups for individual processing, 
-Elastic Machine Groups with dynamic scaling, and MPI Clusters for complex simulations 
-requiring multiple CPUs. Lastly, we present a comparison using a SWASH simulation 
-to demonstrate the performance differences between shared and dedicated resources, 
-providing practical insights into optimizing your simulations with the Inductiva API.
+it comes to making informed decisions about resource allocation. Here, you will 
+learn the two options available for running your simulations; shared and dedicated
+resources. Additionally, we will demonstrate the performance differences between 
+the two resources using a SWASH simulation.
 
 ## Shared Resources
 
 As a standard practice, when you submit simulation tasks through the Inductiva API, 
-they get initially sent to a shared pool of workers on designated Virtual Machines (VMs)
-serving multiple users. These VMs are specifically allocated to facilitate easy 
+they enter a [task queue]() and are sent to a shared pool of workers on designated Virtual Machines (VMs)
+serving multiple users from either the [Google Cloud Provider (GCP)](https://cloud.google.com/compute/docs/machine-resource) or Inductiva's own 
+computational platform (ICE). These VMs are specifically allocated to facilitate easy 
```

Comment:
or, in a future version, Inductiva's...

Reply:
got it

Reply:
I think this meant to be for the Inductiva's own computational platform (ICE) above.